osbenchmark/builder/utils/jvm_resolver.py

```diff
@@ -0,0 +1,103 @@
+import os
+import re
+
+from osbenchmark.exceptions import SystemSetupError
+from osbenchmark.utils import io
+
+
+class JvmResolver:
+    def __init__(self, executor):
+        self.executor = executor
+
+    def resolve_path(self, host, majors):
```

Comment:
I'd rename the func name as ``resolve_jdk_path``(same below) in order to be more specific, because there could be other kinds of paths related to jvm... e.g. log path... or can rename the class as JdkResolver? Otherwise LGTM.

Reply:
That's a good call, jvm doesn't make a ton of sense here. Will change references of `jvm` to `jdk`